src/ResourceManagement/EventHub/Generated/Models/SkuName.cs

```diff
@@ -6,15 +6,17 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 // </auto-generated>
 
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+
 namespace Microsoft.Azure.Management.EventHub.Fluent.Models
 {
 
     /// <summary>
     /// Defines values for SkuName.
     /// </summary>
-    public static class SkuName
+    public class SkuName : ExpandableStringEnum<SkuName>
```

Comment:
In most cases like this we "moved" the class/file under the service's root with an updated comment header so it will not get wiped out if you need to regenerate.

Reply:
agree, will move them to service root.

Reply:
@milismsft done